Applications/ConsumerApi/test/ConsumerApi.Tests.Performance/src/libs/backbone-client/fluent-client.ts

```diff
@@ -0,0 +1,80 @@
+// @ts-expect-error: k6 uses links to packages, which typescript cannot lint.
+import { Httpx } from "https://jslib.k6.io/httpx/0.1.0/index.js";
+
+export class FluentClient {
```

Comment:
This class isn't stateless and therefore isn't thread safe. This will become a problem sooner or later.